.github/workflows/issue_title_check.yaml

```
@@ -0,0 +1,44 @@
+name: Issue Title Check
+
+on:
+  issues:
+    types: [opened, edited]
+
```

could you limit the permission of the workflow? 

```suggestion
permissions:
  issues: read|write
```

---

fixed here https://github.com/envoyproxy/ai-gateway/pull/273/commits/79a03d456118897483b9630f597421cb9c5bd9a3